components/tree/index.jsx

```diff
@@ -53,6 +53,16 @@ class Tree extends React.Component {
 		});
 	}
 
+	shouldComponentUpdate (nextProps, nextState) {
```

Comment:
I know this is kind of weird, but this is the simplest solution that I could think of.  The problem that I was running into was that when clicking on a node, the blur is triggered and it would re-render the tree; as a result, the click event would never be triggered.